tests/test_delete_all_links.py

```diff
@@ -1,16 +1,25 @@
 import requests
+import allure
 from cerberus import Validator
 from lxml import html
 
 
+@allure.feature('Delete service')
```

Comment:
Delete all links handler?

Reply:
Why are you called all this handlers the 'service'?

Reply:
Renamed in ac0daf91b7b49eea8c0a4e2e2a411a3237508000